[WIP] - ZDC - Waveform calibration

cortesep opened the pull request:
Meges all commits of dev_shape

cortesep committed:
Waveform calibration

shahor02 commented:
@cortesep, Please don't use merge to dev, I even could not open PR to your branch. You should set up your git config once via
`git config --global pull.rebase true`
then always do:
```
git checkout dev; git pull;
git checkout your_branch
git rebase dev
```
I've cleaned up your branch, squashed your multiple commits and added fixes for these classes, see the last commit of https://github.com/shahor02/AliceO2/tree/dev_shape_rs, in short, you should not add to root dictionary headers including DPL framework classes.

You better close this PR, fetch my branch via
```
git remoter add shahor02 https://github.com/shahor02/AliceO2.git
git fetch shahor02 --prune
git checkout -b dev_shape_new shahor02/dev_shape_rs
```
apply modifications if needed and open a new PR.

cortesep commented:
Thanks Ruben!
Checked locally and created new pull request.
Ciao,
Pietro

On 5/30/22 23:07, Ruben Shahoyan wrote:
> @cortesep <https://github.com/cortesep>, Please don't use merge to dev, 
> I even could not open PR to your branch. You should set up your git 
> config once via
> |git config --global pull.rebase true|
> then always do:
> 
> |git checkout dev; git pull; git checkout your_branch git rebase dev |
> 
> I've cleaned up your branch, squashed your multiple commits and added 
> fixes for these classes, see the last commit of 
> https://github.com/shahor02/AliceO2/tree/dev_shape_rs 
> <https://github.com/shahor02/AliceO2/tree/dev_shape_rs>, in short, you 
> should not add to root dictionary headers including DPL framework classes.
> 
> You better close this PR, fetch my branch via
> 
> |git remoter add shahor02 https://github.com/shahor02/AliceO2.git git 
> fetch shahor02 --prune git checkout -b dev_shape_new shahor02/dev_shape_rs |
> 
> apply modifications if needed and open a new PR.
> 
> —
> Reply to this email directly, view it on GitHub 
> <https://github.com/AliceO2Group/AliceO2/pull/8985#issuecomment-1141477342>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AN2A5SROQHYICHGAMEOCUVLVMUU25ANCNFSM5XDXBT4A>.
> You are receiving this because you were mentioned.Message ID: 
> ***@***.***>
>
